src/numeric_index.h

```diff
@@ -14,6 +14,11 @@
 
 #define RT_LEAF_CARDINALITY_MAX 500
 
+typedef struct {
+  double value;
+  int appearances;
```

Comment:
size_t. We're trying to stop using `int` for stuff that doesn't actually need to be signed